packages/jobs/lib/execution/action.ts

```diff
@@ -129,7 +129,18 @@ export async function startAction(task: TaskAction): Promise<Result<void>> {
     }
 }
 export async function handleActionSuccess({ taskId, nangoProps, output }: { taskId: string; nangoProps: NangoProps; output: JsonValue }): Promise<void> {
-    await setTaskSuccess({ taskId, output });
+    const task = await setTaskSuccess({ taskId, output });
+
+    const logCtx = logContextGetter.get({ id: nangoProps.activityLogId, accountId: nangoProps.team.id });
+
+    void logCtx.info(`The action was successfully run${formatAttempts(task)}`, {
+        action: nangoProps.syncConfig.sync_name,
+        connection: nangoProps.connectionId,
+        integration: nangoProps.providerConfigKey
+    });
+    void logCtx.success();
+
+    metrics.increment(metrics.Types.ACTION_SUCCESS);
```

Comment:
same for the metrics. I only kept the action duration in the controller since it measure the full action call duration